app/src/main/cpp/utils/log.cpp

```diff
@@ -15,67 +15,60 @@
  */
 
 #include <android/log.h>
+#include <errno.h>
+#include <string.h>
```

Comment:
Is there a particular reason why you don't use the modern C++ header names for errno.h and string.h?

```suggestion
#include <cerrno>
#include <cstring>
```